### PR TITLE
Adding npm installation guide

### PR DESCRIPTION
**Pull request by connorjburton**

### CODE REVIEW

**Comment by hhsnopek** [comment]
Hey @connorjburton would you be able to rebase master onto this?

File: README.md [hunk 1]
```diff
@@ -49,6 +49,30 @@ new ShareButton({
...
+<share-button></share-button>
+```
+
+```js
+var ShareButton = require('share-button');
+
+new ShareButton({
+  networks: {
+    facebook: {
+      app_id: "abc123"
```
Comment (hhsnopek) [inline README.md]: only reason for the rebase is to change `app_id` to `appId`

**Comment by connorjburton** [comment]
Should be done

**Comment by hhsnopek** [comment]
uh oh, looks like your tree got separated from our master - the last commit, `Merge branch 'npm-...'` shouldn't exist because it hasn't been merged into our share-button master. did you perform `git pull upstream master -r`? upstream should be pointing at `carrot/share-button`

**Comment by connorjburton** [comment]
Uh oh indeed, sorry, hopefully #289 is better

**Pull request closed without merging**